docs/docs/legal/data-processing-agreement.md

```diff
@@ -0,0 +1,229 @@
+
+This Annex to the Agreement governs the Parties&#39; data protection obligations in addition to the provisions of the Agreement.
+
+## Subject matter, duration, nature and purpose of the processing as well as type of personal data and categories of data subjects
+
+This annex reflects the commitment of both parties to abide by the applicable data protection laws for the processing of Personal Data for the purpose of Processor's execution of the Framework Agreement. 
+
+The duration of the Processing shall correspond to the duration of the Agreement, unless otherwise provided for in this Annex or unless individual provisions obviously result in obligations going beyond this.
+
+In particular, the following Personal Data are part of the processing:
+<table>
+  <tr>
+    <th>Type of personal data</th>
+    <th>Examples</th>
+    <th>Affected data subjects</th>
+  </tr>
+  <tr>
+    <td><strong>Basic data</strong></td>
+    <td>
+      <ul>
+        <li>Surname and first name</li>
+        <li>Email addresses</li>
+        <li>User name</li>
+        <li>Language</li>
+      </ul>
+    </td>
+    <td>All users</td>
+  </tr>
+  <tr>
+    <td><strong>Login data</strong></td>
+    <td>
+      <ul>
+        <li>Randomly generated ID</li>
+        <li>Password</li>
+        <li>Public keys / certificates (&quot;FIDO2&quot;, &quot;U2F&quot;, &quot;x509&quot;, ...)</li>
+        <li>User names or identifiers of external login providers</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>
+      <p>All users</p>
+      <p>Password: Users who use authentication methods with password.</p>
+      <p>Public Keys: Users who use an authentication procedure with cryptographic keys.</p>
+      <p> External login provider identifiers: Users who use an external login provider.</p>
+      <p>Phone number: Users who use authentication methods with SMS</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Profile data</strong></td>
+    <td>
+      <ul>
+        <li>Profile pictures</li>
+        <li>Gender</li>
+        <li>Birthday</li>
+        <li>Language</li>
+        <li>Address(es)</li>
+        <li>Phone number(s)</li>
+      </ul>
+    </td>
+    <td>Users who voluntarily add profile data</td>
+  </tr>
+  <tr>
+    <td><strong>Communication data</strong></td>
+    <td>
+      <ul>
+        <li>Emails</li>
+        <li>Chats</li>
+        <li>Call metadata</li>
+      </ul>
+    </td>
+    <td>Customers and users who communicate with us directly (e.g. support)</td>
+  </tr>
+  <tr>
+    <td><strong>Payment data</strong></td>
+    <td>
+      <ul>
+        <li>Billing address</li>
+        <li>Customer number</li>
+        <li>Customer history</li>
+        <li>Credit rating information</li>
+      </ul>
+    </td>
+    <td>
+      <p>Customers who use services that require payment</p>
+      <p>Credit rating information: Only customers who pay by invoice</p>
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Data traffic data</strong></td>
```

Comment:
```suggestion
    <td><strong>Meta data</strong></td>
```

Reply:
Or we could say something like "usage related data"

Reply:
How about "Traffic meta data"?

Reply:
Well its not only related to "traffic" it's more related to "usage"